authoring/docker-compose.solr.yml

```diff
@@ -19,7 +19,8 @@ services:
     ports:
       - 8694:8983
     environment:
-      - "SOLR_OPTS=-Xss1024K -Xmx1G"
+      - "SOLR_JAVA_MEM=-Xmx1G"
+      - "SOLR_JAVA_STACK_SIZE=-Xss1024K"
```

Comment:
Do we need an empty SOLR_JVM_OPTS or not?

Reply:
@avasquez614 ^

Reply:
agree